webapp/channels/src/components/update_user_group_modal/update_user_group_modal.tsx

```diff
@@ -162,9 +162,9 @@ const UpdateUserGroupModal = (props: Props) => {
                     aria-label='Close'
                     onClick={goBack}
                 >
-                    <LocalizedIcon
+                    <i
                         className='icon icon-arrow-left'
-                        ariaLabel={{id: t('user_groups_modal.goBackLabel'), defaultMessage: 'Back'}}
+                        aria-label={formatMessage({id: 'user_groups_modal.goBackLabel', defaultMessage: 'Back'})}
```

Comment:
Could you add the ARAI label in the button & remove it from this icon.

Reply:
done